chemprop/cli/train.py

```diff
@@ -827,7 +827,11 @@ def train_model(
 
         try:
             trainer_logger = TensorBoardLogger(model_output_dir, "trainer_logs")
-        except ModuleNotFoundError:
+        except ModuleNotFoundError as e:
+            logger.warning(
+                "Unable to import TensorBoardLogger, reverting to CSVLogger (original error: %s).",
+                str(e),
+            )
```

Comment:
```suggestion
            logger.warning("Unable to import TensorBoardLogger, reverting to CSVLogger...")
            logger.warning(f"Original error: %s")
```

Reply:
I think the `str(e)` is still needed here.

Reply:
messed up my `f`-string.

Reply:
I prefer to keep these in a single warning

Reply:
You should still use an f-string.